mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -1243,7 +1247,139 @@
 							"    tv4.addSchema(\"schema_hasManyRelationship.json\", JSON.parse(pm.variables.get(\"schema_hasManyRelationship\")));",
 							"    tv4.addSchema(\"schema_hasOneRelationship.json\", JSON.parse(pm.variables.get(\"schema_hasOneRelationship\")));",
 							"    tv4.addSchema(\"schema_included.json\", JSON.parse(pm.variables.get(\"schema_included\")));",
-							"    tv4.addSchema(\"schema_jsonapi.json\", JSON.parse(pm.variables.get(\"schema_jsonapi\")));"
+							"    tv4.addSchema(\"schema_jsonapi.json\", JSON.parse(pm.variables.get(\"schema_jsonapi\")));",
+							"    tv4.addSchema(\"schema_tags.schema\", JSON.parse(pm.variables.get(\"schema_tags\")));"
+						]
+					}
+				}
+			]
+		},
+		{
+			"name": "setup for providers adding tags",
```

Comment:
Just wondering why this `setup for provider with adding tags`  is needed? Its looks like we are testing tags as part of PUT request (both adding and removing tags as part of these tests) - the tags added in this setup step are not checked for anywhere. And because there is a setup step, I was looking for a matching teardown step.

Reply:
We are not able to create and delete new provider. And we have not had providers with existing tags. As setup step I use updating existing provider by adding new tags.
We do not have teardown step because tags is new functionality and it should be added to providers as a new attribute of this object.

Reply:
Thanks Stan,  I understand all this -- my question/comment was that I  don't see anywhere in the tests where you are using and testing for foo-tag that that was added in this setup step -- that is why I am questioning  it. For example  - with `PUT update provider with tags` you set tags and you check for them and then there is another test that tests an empty tag list . Which test is using the setup?

Reply:
Thanks Carole,it makes sense to test adding tags in setup step.

Reply:
Thanks Stas -- I think this is fine -- I was questioning the setup step because it didn't look like the setup for providers tags was used elsewhere by other tests